### DISCUSSION TITLE
Leather Horse Armor Color

### DISCUSSION THREAD

**Discussion by vearendev** [Minestom/Minestom#1620] [Help] [answered]
Is there a way to change the color of leather horse armor?

**Comment by Flopgop** [comment, marked as answer]
I actually had to do this the other day, you can do something like this
```java
int color = ... // your color
ItemStack armor = ... // your horse armor itemstack
ItemStack coloredArmor = armor.withMeta(bu -> bu.setTag(Tag.NBT("display"), NBT.Compound(bui -> bui.put("color", NBT.Int(color)));
```

**Reply by FrederikHeinrich** [reply]
https://github.com/FrederikHeinrich/Minestom/commit/df84c200f3d6a4d98d42a320af8316f943c550ad have fun :)

**Reply by Flopgop** [reply]
oh well that would have been a lot easier

**Reply by emortaldev** [reply]
> [FrederikHeinrich@df84c20](https://github.com/FrederikHeinrich/Minestom/commit/df84c200f3d6a4d98d42a320af8316f943c550ad) have fun :)

nooo :crying_cat_face: please use HSV instead of RGB

**Reply by FrederikHeinrich** [reply]
if you know how to handle HSV you dont need the example code ;)
